tests/testthat/test-raoq.R

```diff
@@ -52,6 +52,39 @@ test_that("Rao's Quadratic Entropy works on subset of site/species", {
                         "and site-species matrix\nTaking subset of species"))
 })
 
+test_that("Rao's Quadratic Entropy works on sparse matrices", {
+
+  library("Matrix")
+
+  site_sp <- matrix(1, ncol = nrow(traits_birds))
+  colnames(site_sp) <-  rownames(traits_birds)
+  rownames(site_sp) <- "s1"
+
+  sparse_site_sp <- as(site_sp, "sparseMatrix")
```

Comment:
The clean way to do this according to the docs seems to be:
```suggestion
  sparse_site_sp <- Matrix(site_sp, sparse = TRUE)
```

Reply:
Actually we can also accept dense matrix objects from the `Matrix` package